### PR TITLE
Fix: typo for recipe JSON files

### PR DESCRIPTION
**Pull request by Digitalisx**
Hello, everyone in the community! 🙂
I founded some typo while analyze code, So I fixed it.
If you have any additional comments or feedback, please leave a thread!

### CODE REVIEW

File: data/recipes/grr_artifact_ts.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
         ["--approvers", "Emails for GRR approval request.", null],
         ["--sketch_id", "Timesketch sketch to which the timeline should be added.", null],
         ["--wait_for_timelines", "Whether to wait for Timesketch to finish processing all timelines.", true],
-        ["--analyzers", "Timesketch analysers to run", null],
+        ["--analyzers", "Timesketch analyzers to run", null],
```
Comment (ramo-j) [inline data/recipes/grr_artifact_ts.json:48]: `analysers` is not a typo, it's UK English spelling.

**Comment by ramo-j** [comment]
Thanks for the contributions @Digitalisx! Just a note, this project is maintained by folks all around the world, and so British English spelling shows up from time to time, and so `analysers` is not a typo.

File: data/recipes/grr_artifact_ts.json (hunk 1 above)
Comment (Digitalisx) [reply data/recipes/grr_artifact_ts.json:48]: You're right. Just notation is different, not a typo!
However, my intention to amend that was "unification".
The option was labeled `--analyzers`, but the corresponding description was `analysers`.
Please tell me your thoughts on revising with these intentions! I'll reflect it right away.

**Comment by Digitalisx** [comment]
@ramo-j, Thank you for your review!
I left my opinion on the comments you left.

File: data/recipes/grr_artifact_ts.json (hunk 1 above)
Comment (ramo-j) [reply data/recipes/grr_artifact_ts.json:48]: Ahh excellent point, we should keep it the same.

**Review by ramo-j — APPROVED** [review]

**Pull request merged**